docs/Threading.md

```diff
@@ -0,0 +1,167 @@
+The SGX-LKL threading implementation
```

Comment:
```suggestion
# The SGX-LKL threading implementation
```

Reply:
I don't think we need two kinds of markup to say that this is h1.